bears/ruby/RuboCopBear.py

```diff
@@ -9,6 +9,16 @@
 from coalib.results.Result import Result
 
 
+def map_naming_convention_to_rubocop(setting):
+    '''
+    This function converts the string ``setting`` (that refers to a naming
+    convention) into a ``rubocop`` specific naming convention.
+    '''
+    naming_convention = {'camel': 'camelCase', 'snake': 'snake_case'}
```

Comment:
no need for a function, just use `naming_convention.get(setting, "snake_case")` and dump this function